faf.spec.in

```diff
@@ -28,6 +28,16 @@ Requires: python-psycopg2 >= 2.5
 Requires: python-sqlalchemy >= 0.8
 Requires: rpm-python
 
+# six library to cover Python 3 compatibility
+%if 0%{fedora}
```

Comment:
It should be `0%{?fedora}`. Sorry I had it incorrect in my previous comment as well.